Linkedin_Connections_Scrapper/script.py

```diff
@@ -0,0 +1,198 @@
+# Linkedin My_Connections Scrapper
+# Written by XZANATOL
+from selenium.webdriver.common.action_chains import ActionChains
+from optparse import OptionParser
+from selenium import webdriver
+import pandas as pd
+import time
+import re
+
+pattern_name = "\\n(.+)\\n"  # Used to extract names
+pattern_headline = 'occupation\\n(.+)\\n'  # Used to extract headlines
+
+# Help menu
+usage = """
+<Script> [Options]
+
+[Options]
+    -h, --help        Show this help message and exit.
+    -e, --email       Enter login email
+    -p, --password    Enter login password
+    -s, --skills      Flag to scrap each profile, and look at its skill set
+
+Operation Modes:
+> Basic mode
+    This will scrap all LinkedIn connections list with there corresponding Name, Headline, and Profile link.
+> Skills scrapper mode (-s/--skills)
+    (Time Consuming mode)
+    This will do the same job of basic mode but along with visiting each profile and extracting the skills of each.
+"""
+
+# Load args
+parser = OptionParser()
+parser.add_option("-e", "--email", dest="email", help="Enter login email")
+parser.add_option("-p", "--password", dest="password", help="Enter login password")
+parser.add_option("-s", "--skills", action="store_true", dest="skills", help="Flag to scrap each profile, and look at its skill set")
+
+
+def login(email, password):
```

Comment:
Please add some logic to verify if user was logged in or not. You can do that by either looking for the error message or by checking the page title by using `driver.title`